store/store.go

```diff
@@ -60,6 +62,7 @@ const (
 	// communicating with the store. History:
 	//  - "1": client supports squashfs snaps
 	UbuntuCoreWireProtocol = "1"
+	MaxRetryDuration       = time.Duration(30) * time.Second
```

Comment:
This looks like a red flag to me. If this is a const either there is something you're not testing, or there are tests that take more than 30 seconds.